### PR TITLE
Sf3 compatbility fix for sync-thumbnails command

### PR DESCRIPTION
**Pull request by acrobat**
I am targeting this branch, because of sf3 compatibility fix.

## Changelog

```markdown
### Fixed
- Sf3 compatibility on the sync-thumbnails command (dialog helper)

```

## Subject

We had some issues with our images after adding a new image format, but we can't run the command as we use sf 3.3

### CODE REVIEW

File: Command/SyncThumbsCommand.php [hunk 1]
```diff
@@ -82,9 +97,9 @@ public function execute(InputInterface $input, OutputInterface $output)
         $fsRegister->setAccessible(true);
 
         $batchCounter = 0;
-        $batchSize = intval($input->getOption('batchSize'));
-        $batchesLimit = intval($input->getOption('batchesLimit'));
-        $startOffset = intval($input->getOption('startOffset'));
+        $batchSize = (int) $input->getOption('batchSize');
+        $batchesLimit = (int) $input->getOption('batchesLimit');
+        $startOffset = (int) $input->getOption('startOffset');
```
Comment (acrobat) [inline Command/SyncThumbsCommand.php]: `(int)` casting is faster than `intval()`

**Comment by acrobat** [comment]
Is it possible to have a review (and possibly merge) because we're stuck on this command with sf3.3.

/cc @greg0ire @core23 @Soullivaneuh

**Comment by jordisala1991** [comment]
Thanks @acrobat

**Comment by greg0ire** [comment]
One PR with one commit please. Fix one problem in one commit, there is no reason to fix it by command, file or method.

**Pull request merged**